modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
```

Comment:
Member Roll should probably be capitalized, because the product name is also capitalized.

Reply:
The product name has to be capitalized, but we use sentence case headings at Red Hat not title case headings.